### PR TITLE
feat: [PIE-7488]: Swagger update

### PR DESCRIPTION
**Pull request by Eric761**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by sonaligoyal-harness — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx [hunk 1]
```diff
@@ -98,13 +98,14 @@ const checkDuplicateStep = (
...
 export const updateStepWithinStage = (
-  execution: ExecutionElementConfig,
+  execution: ExecutionElementConfig | StepGroupElementConfig,
   processingNodeIdentifier: string,
   processedNode: StepElementConfig | TemplateStepNode,
   isRollback: boolean
 ): void => {
   // Finds the step in the stage, and updates with the processed node
-  execution?.[isRollback ? 'rollbackSteps' : 'steps']?.forEach(stepWithinStage => {
+  const _execution = isRollback ? (execution as ExecutionElementConfig)?.rollbackSteps : execution?.steps
```
Comment (samarth-sharma31) [inline src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx:107]: _execution can be renamed to executionSteps (to increase readability) ---> now executionSteps can either be rollbackSteps (if isRollback is true else steps) and then we can iterate on executionSteps

const executionSteps = get(execution, isRollback ? 'rollbackSteps' : 'steps') as ExecutionWrapperConfig[]

and then we can use executionSteps further in line 108

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx [hunk 2]
```diff
@@ -129,7 +127,6 @@ export const ActiveServiceInstancePopover: React.FC<ActiveServiceInstancePopover
     buildIds: [buildId],
     clusterIdentifier: clusterId,
     infraIdentifier,
-    lastDeployedAt,
```
Comment (samarth-sharma31) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx:132]: why are we removing this in this PR? is this intentional ?

Comment (Eric761) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancePopover.tsx:132]: Yes, this is intentional.

File: src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx (hunk 1 above)
Comment (Eric761) [reply src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx]: Made the change.

**Review by vikrant-harness — APPROVED** [review]

**Review by sonaligoyal-harness — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Pull request merged**